### ISSUE TITLE
ssl configure error 

### ISSUE THREAD

**Issue by aixj1984** [Kozea/Radicale#494]
1. one word error: ssl use ‘ciphers’ param name, bug the project use 'cyphers'
2. run time error:
```
2016-08-29 17:34:33,943 - ERROR: An exception occurred during server startup:
Traceback (most recent call last):
  File "/opt/test/Radicale/radicale/__main__.py", line 108, in run
    serve(configuration, logger)
  File "/opt/test/Radicale/radicale/__main__.py", line 193, in serve
    address, port, application, server_class, RequestHandler)
  File "/usr/lib/python3.5/wsgiref/simple_server.py", line 160, in make_server
    server = server_class((host, port), handler_class)
  File "/opt/test/Radicale/radicale/__init__.py", line 113, in __init__
    ssl_version=self.protocol, ciphers=self.ciphers)
  File "/usr/lib/python3.5/ssl.py", line 1069, in wrap_socket
    ciphers=ciphers)
  File "/usr/lib/python3.5/ssl.py", line 686, in __init__
    self._context = SSLContext(ssl_version)
  File "/usr/lib/python3.5/ssl.py", line 361, in __new__
    self = _SSLContext.__new__(cls, protocol)
TypeError: an integer is required (got type NoneType)
2016-08-29 17:34:33,944 - DEBUG: Cleaning up
```

**Comment by liZe** [comment]
Thanks for the report!

**Comment by aixj1984** [comment]
how about the second problem?

**Comment by liZe** [comment]
The first problem is fixed by fd1742f and the second one by 947c573.
